### PR TITLE
Complement of the locale fr

### PR DESCRIPTION
**Pull request by Bundy01**
Addition of complement for the translation fr.

### CODE REVIEW

**Comment by Bundy01** [comment]
Sorry,
I noticed that in French there is a space before a ' :'. Please take this into consideration when making changes:)
Thanks.

File: locale/fr.po [hunk 1]
```diff
@@ -237,7 +237,7 @@ msgstr "[a] annuler"
 
 #: pikaur/install_cli.py:559
 msgid "[c] git checkout -- '*'"
-msgstr "[c] git checkout -- '*'"
```
Comment (actionless) [inline locale/fr.po]: that's a command...

**Comment by actionless** [comment]
> I noticed that in French there is a space before a ' :'.

you mean in french it should be a space between a word and `:`?

**Comment by Bundy01** [comment]
In locale/fr.po:

 > @@ -237,7 +237,7 @@ msgstr "[a] annuler"

 #: pikaur/install_cli.py:559
 msgid "[c] git checkout -- '*'"
-msgstr "[c] git checkout -- '*'"
that's a command...



Ok, sorry



you mean in french it should be a space between a word and :?

Yes, it's French punctuation :/
> 

Le dim. 3 juin 2018 à 11:05, Yauhen Kirylau <notifications@github.com> 
a écrit :
>> I noticed that in French there is a space before a ' :'.
>> 
you mean in french it should be a space between a word and :?

**Comment by actionless** [comment]
> Yes, it's French punctuation :/

why then did you removed the spaces there in your PR?

**Comment by Bundy01** [comment]
It was a mistake on my part, I thought there shouldn't be spaces.

**Comment by actionless** [comment]
then please update the pr accordingly

**Comment by IgnusG** [comment]
You don’t have to close a PR and open a new one if you just want to edit commits within. Just do a `git push —force`

**Pull request closed without merging**